scripts/test_e2e.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -ex
+
+export BROWSERSTACK_USERNAME=andrewwylde1
```

Comment:
Should this change?

You can do:

```
BROWSERSTACK_USERNAME=${BROWSERSTACK_USERNAME:=andrewwylde1OrOtherDefault}
```

Which will use the inherited envs BROWSERSTACK_USERNAME, if present, defaulting to `andrewwylde1OrOtherDefault`